### PR TITLE
Replaced PrBodyContentReply `n with array

### PR DESCRIPTION
**Pull request by OfficialEsco**
I just learned the use of @() 🤦‍♂️
Needed the space on line 983 to make the PR look less cluttered

### CODE REVIEW

**Comment by Trenly** [comment]
@OfficialEsco I’m wondering if we can abstract the switch a bit further now that we have the standardized menus. 

Soemthing like:
```raw
Function
  For each line
    Switch -Wildcard
        Set the menu options, don’t display menu
    End switch
    Display menu and replace line
  Next line
End function
```

The more I think about it though, the more I believe the KeypressMenu function would need to be adjusted to handle empty strings. I can handle this in a separate PR

**Comment by OfficialEsco** [comment]
Think its best if you just show it in action, usually they make a lot more sense 😁

File: Tools/YamlCreate.ps1 [hunk 1]
```diff
@@ -980,14 +980,14 @@ Function Enter-PR-Parameters {
             Write-Host
             Write-Host "Enter issue number. For example`: 21983, 43509"
             $ResolvedIssues = Read-Host -Prompt 'Resolved Issues'
+            $PrBodyContentReply += @("")
             Foreach ($i in ($ResolvedIssues.Split(",").Trim())) {
-                $PrBodyContentReply += "Resolves #$i`n"
```
Comment (Trenly) [inline Tools/YamlCreate.ps1:984]: I know its a bit outside the scope of your changes, but is there any way for us to validate the issue exists? Maybe using Invoke-WebRequest and checking the return status code?

Also, I know this isn’t your fault, but the way the issues are handled makes it so if someone entered `microsoft/winget-cli/#1234` or `#1234` it wouldn’t properly link the resolves. I think we need to have separate logic handling for if the `$i` has a # in it already

**Review by Trenly — APPROVED** [review]
All comments are just minor suggestions and could be handled in a separate PR

File: Tools/YamlCreate.ps1 (hunk 1 above)
Comment (OfficialEsco) [reply Tools/YamlCreate.ps1:984]: Off topic: i have/had a issue when fetching some patterns from the Schema, for example i would like the Pattern from PackageVersion which is `^[^\\\\/:\\*\\?\"<>\\|\\x01-\\x1f]+$` however the output turned into something like `^[^\/:\*\?\"<>\|\x01-\x1f]+$` which obviously did not work

Do you have a workaround for this?

Comment (Trenly) [reply Tools/YamlCreate.ps1:984]: Do you have specific line numbers that I can look at?

Comment (OfficialEsco) [reply Tools/YamlCreate.ps1:984]: Line 176 i want 
`-and ($script:CopyrightUrl -notmatch $InstallerSchema.definitions.PackageVersion.pattern)`

You can test this
```
$InstallerSchema = @(Invoke-WebRequest 'https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.0.0/manifest.installer.1.0.0.json' -UseBasicParsing | ConvertFrom-Json)
$InstallerSchema.definitions.PackageVersion.pattern
```
Output: `^[^\\/:\*\?"<>\|\x01-\x1f]+$` expected `^[^\\\\/:\\*\\?\"<>\\|\\x01-\\x1f]+$`

**Pull request merged**